mycpp/tuple_types.h

```diff
@@ -1,11 +1,19 @@
 #ifndef TUPLE_TYPES_H
 #define TUPLE_TYPES_H
 
+
+
 template <class A, class B>
 class Tuple2 {
  public:
-  Tuple2(A a, B b) : a_(a), b_(b) {
+  Tuple2(A a, B b) : a_(a), b_(b)
+  {
+    heap_tag_ = Tag::FixedSize;
```

Comment:
I think it's better to use the `Obj()` constructor like Dict does

So then the body only has the `|=`

Reply:
If you can figure out how to get the compiler to accept Tuple inheriting from Obj, be my guest.  I tried to get it to cooperate and it would not